src/app/entry.service.ts

```diff
@@ -1,19 +1,25 @@
 import { Injectable } from '@angular/core';
 import {HttpClient} from '@angular/common/http'
+import { Subject } from 'rxjs';
+import {url_config} from '../config/config'
```

Comment:
inconsistent variable name
https://angular.io/guide/styleguide

use lower camel case for variables 

https://indepth.dev/posts/1181/angular-cli-camelcase-or-kebab-case-2

Reply:
please check in whole code